pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -412,7 +513,10 @@ async def client_streams(
             yield read_stream, write_stream
 
     def __repr__(self) -> str:
-        return f'MCPServerStdio(command={self.command!r}, args={self.args!r}, tool_prefix={self.tool_prefix!r})'
+        if self.id:
+            return f'{self.__class__.__name__} {self.id!r}'
```

Comment:
This feels wrong, at least it should be wrapped in something that makes it clear where it ends, e.g.:
```suggestion
            return f'<{self.__class__.__name__} {self.id!r}>'
```
but presumably the dataclasses-esque rendering below is closer to what we want